src/completion.ts

```diff
@@ -17,6 +17,7 @@ import { ReplicateHandler } from './handlers/replicate';
 
 export const MODEL_HANDLER_MAPPINGS: Record<string, Handler> = {
   'claude-2': AnthropicHandler,
```

Comment:
```suggestion
  'claude-': AnthropicHandler,
```
As we do regex matches now, this could work too. But please add a test in `getHandler.test`